main.py

```diff
@@ -57,7 +58,7 @@ def respond(voice_data):
         webbrowser.get().open(url)
         alina_speak(f"Here is what I found for {search}")
     if "what is your name" in voice_data:
-        alina_speak("My name is Tejax.")
+        alina_speak("My name is alina.")
```

Comment:
You've created a new variable assistant_name and added the feature to change it using the voice, but you forgot to replace the name from the string and using the assistant_name variable here instead.

Reply:
commit this small change and I'll merge the Pull Request right away